MaterialSamples/src/main/java/io/doist/material/sample/ElevatedTextView.java

```diff
@@ -0,0 +1,137 @@
+package io.doist.material.sample;
+
+import android.content.Context;
+import android.graphics.drawable.Drawable;
+import android.util.AttributeSet;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import io.doist.material.elevation.ElevationDelegate;
+
+public class ElevatedTextView extends TextView implements ElevationDelegate.Host {
```

Comment:
This needs to be updated, it's broken after the last refactoring. But it ends up being very nice and simple! :)